addons/wp/stringtable.xml

```diff
@@ -207,6 +209,12 @@
             <English>Task Garrison makes the unit take up positions in buildings and man available turrets and static weapons within an area defined by 3DEN attributes.</English>
             <German>Task Besetzen lässt Einheiten innerhalb des Gebietes das durch die Eden Editor Attribute festgelegt wurde, Stellung in Häusern beziehen und Geschütze bemannen.</German>
         </Key>
+        <Key ID="STR_Lambs_WP_Module_TaskGarrison_Patrol_DisplayName">
+            <English>Dynamic patrol</English>
+        </Key>
+        <Key ID="STR_Lambs_WP_Module_TaskGarrison_Patrol_ToolTip">
+            <English>The unit will subdivide a small sentry group to patrol the within the two times the radius</English>
```

Comment:
this should have a Warning included that the Groups breakup and is with that More Destructive

Reply:
Willco